docs/docs/workflows/workspaces/README.md

```diff
@@ -0,0 +1,63 @@
+# Workspaces
+
+When registering a Pipedream account, you're prompted to create a workspace. Workflows are organized and contained within your workspace.
+
+Workspaces are a container for your projects and workflows, and can be shared with your team members.
+
+<div class="flex justify-center">
+  <img src="https://res.cloudinary.com/pipedreamin/image/upload/v1674062859/docs/CleanShot_2023-01-18_at_12.27.13_itqa7f.png" alt="How workspaces are organized" />
```

Comment:
This diagram was a little confusing to me. Maybe have one workspace at the top, branching off into two projects, each branching off into two workflows? Like a [family tree diagram](https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.pinterest.com%2Fpin%2F373517362845133814%2F&psig=AOvVaw3AhlnYBcKW_TKGFdLhYa8Y&ust=1674159734807000&source=images&cd=vfe&ved=0CA8QjRxqFwoTCKC_nJj50fwCFQAAAAAdAAAAABAD). Then you immediately visualize the one-to-many relationship between all objects.

Reply:
I'll need access to another tool for diagramming, unfortunately Mermaid.js in Notion doesn't have that kind of diagram control.

Reply:
I've photopea'd something better. Will go over with Neil to make sure it's in his brand guidelines

https://res.cloudinary.com/pipedreamin/image/upload/v1674143831/docs/New_Project_5_zep4ii.png